src/morale_types.h

```diff
@@ -5,6 +5,7 @@ enum morale_type : int {
     MORALE_NULL = 0,
     MORALE_FOOD_GOOD,
     MORALE_FOOD_HOT,
+    MORALE_FOOD_CARNIVORE,
```

Comment:
New enum values always have to be added second-to-last, after last enum value, but before the `NUM_` value.

Though it would be a good idea to redo morale bonuses as json-loaded objects instead of those ugly enums.